diff --git BUILD.bazel BUILD.bazel
--- BUILD.bazel
+++ BUILD.bazel
@@ -1 +1,5 @@
+load("@protobuf//bazel:cc_proto_library.bzl", "cc_proto_library")
+load("@protobuf//bazel:proto_library.bzl", "proto_library")
+load("@rules_cc//cc:defs.bzl", "cc_library", "cc_test")
+
 package(default_visibility = ["//visibility:public"])
diff --git MODULE.bazel MODULE.bazel
--- MODULE.bazel
+++ MODULE.bazel
@@ -8 +8,2 @@
 bazel_dep(name = "protobuf", version = "31.1")
+bazel_dep(name = "rules_cc", version = "0.2.17")
